angular/projects/public-lng/src/assets/styles/components/leaflet.scss

```diff
@@ -147,4 +195,26 @@ margin-bottom:10px;
   .leaflet-custom-minimap {
     display: none;
   }
+
+  .map-popup-content-fixed .leaflet-popup-content{
+    width: auto !important;
+  }
+
+  .leaflet-popup.map-popup-content-fixed {
+    bottom:auto !important ;
+    left: 1rem !important ;
+    right: 1rem !important ;
+    width: auto !important;
+  }
+
+  .leaflet-control-zoom{
+    display: none;
+  }
+
+  
+#map .popup-desc {
```

Comment:
Please run code formatter on this file.  There are quite a few places with inconsistent indentation and brace spacing